src/common/SurgeStorage.cpp

```diff
@@ -1804,13 +1785,7 @@ void SurgeStorage::storeMidiMappingToName(std::string name)
    doc.InsertEndChild( sm );
 
    fs::create_directories( userMidiMappingsPath );
-   std::string fn = userMidiMappingsPath +
-#if WINDOWS
-      "\\"
-#else
-      "/"
-#endif
-      + name + ".srgmid";
+   std::string fn = Surge::Storage::appendDirectory(userMidiMappingsPath, name, ".srgmid");
```

Comment:
This is wrong - will make foo/name/.srgmid. You need to add not use the 2 arg version

Reply:
Oops ^_^;